### ISSUE TITLE
Teacher message passing

### ISSUE THREAD

**Issue by krashanoff** [uclaacm/teach-la-go-backend#43]
Teachers should be able to request to view the latest version of their student's work, and receive live updates. To do this, they should send a message of the form:

```
{
  "type": "READ",
  "target": "some_uid"
}
```

This will cause the backend to enter a state of propagating packets from the given `target` to the owner, and forwarding packets from the `owner` to the given `target`. For example, consider the following pseudocode:

```go
out := read(someSocket)

if someSocket == owner {
    // forward all packets from owner to target.
    // receive packets from target, forward to owner.
}
```

This issue depends on #42, and may spawn another issue to abstract client connections on top of Go's websockets (for improved routing behavior).
